guides/06_FAQs.md

```diff
@@ -34,15 +34,39 @@ If you have feature requests or feedback, please fill out the [customer question
 
 The meetings created by the Amazon Chime Application and the SDK are distinct. Amazon Chime SDK is meant for customers who would like to build the experience into their existing applications and it is **NOT** a means to create an Amazon Chime meeting (with a PIN) that can be joined using our client application. You cannot use the Amazon Chime SDK to build a custom client to join an Amazon Chime meeting with a PIN.
 
+### Does the MeetingsNotificationsConfiguration only support SQS queues in us-east-1?
+
+Yes, we currently support SNS topics or SQS queues located in us-east-1 region only. 
+
+### How can I learn about interruptions to the Amazon Chime service?
+
+You can be notified about Amazon Chime service interruptions at the https://status.aws.amazon.com/ website. In addition you can set up [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) events via the [AWS Personal Health Dashboard](https://aws.amazon.com/premiumsupport/technology/personal-health-dashboard/) for the Amazon Chime service. You can find the Amazon Chime SLA on this [webpage](https://aws.amazon.com/chime/sla/).
+
 ## Browser support
 
 ### What browsers are supported by the Amazon Chime SDK for JavaScript?
 
 You can find the complete list of browsers in this link: [Supported Browsers](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-browsers). [WebRTC](https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API) support in a browser is a prerequisite for Amazon Chime SDK to run. Browsers that do not offer WebRTC support, like Internet Explorer, are not supported.
 
-### How can I learn about interruptions to the Amazon Chime service?
+### I am unable to select audio output device in some browsers, is this a known issue?
```

Comment:
`### I am unable to select an audio output device in some browsers, is this a known issue?`